1.10.2/src/main/java/fr/ourten/brokkgui/wrapper/BrokkGuiWrapperMod.java

```diff
@@ -0,0 +1,28 @@
+package fr.ourten.brokkgui.wrapper;
+
+import fr.ourten.brokkgui.BrokkGuiPlatform;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+
+/**
+ * @author Ourten 5 oct. 2016
+ */
+@Mod(modid = BrokkGuiWrapperMod.MODID, version = BrokkGuiWrapperMod.VERSION, name = BrokkGuiWrapperMod.MODNAME)
+public class BrokkGuiWrapperMod
+{
+    public static final String MODID   = "brokkguiwrapper";
+    public static final String MODNAME = "BrokkGui Wrapper";
+    public static final String VERSION = "0.1.0";
+
+    @EventHandler
+    public void onPreInit(final FMLPreInitializationEvent event)
+    {
+        BrokkGuiPlatform.getInstance().setPlatformName("MC1.7.10");
```

Comment:
This wrapper must use this platformName : MC1.10.2, perhaps in the future we will let the version be inferred from a variable, but let's keep this as a constant for now.